tcpdf.php

```diff
@@ -16918,7 +16918,7 @@ protected function getHtmlDomArray($html) {
 							$dom[$key]['height'] = $dom[$key]['style']['height'];
 						}
 						// check for text alignment
-						if (isset($dom[$key]['style']['text-align'])) {
+						if (isset($dom[$key]['style']['text-align'][0])) {
```

Comment:
```suggestion
						if (isset($dom[$key]['style']['text-align']) && isset($dom[$key]['style']['text-align'][0])) {
```
Will this avoid a crash if the text align key is missing ?

Reply:
Yep: https://3v4l.org/gVpPk

Reply:
Awesome, thanks !
I checked for EOL versions too